patterns/1-initial/overcoming-project-management-time-pressures.md

```diff
@@ -16,26 +16,32 @@ Customer deadlines and delivery commitments for feature content do not change.
 
 ## Forces
 
-- Aggressive roadmaps for delivery
-- Project management concern that engagement will lead to missed deadlines
-- Project management concern that Code contribution and/or mentoring may lead to dilution of subject matter expert time spent on their own projects (other teams may require their time).
+❎ Aggressive roadmaps for delivery
```

Comment:
Why were these icons introduced?
There are also other ones (checkmarks) in rows 26-32.

I am not against using these but haven't used them elsewhere so would be good to know what they mean :)

Reply:
Reading these again it looks like these forces were grouped into 2 categories?
- the first 3 bullets negativ
- the next 4 bullets positive

The `-` and `+` sign used to express this grouping previously does not get rendered like this in markdown.

So it seems like the new visualization here tries to fix this by highlighting this grouping with symbols.

Reply:
Will leave this as is, now that I think I understood the intention :)